src/pages/friend.js

```diff
@@ -40,160 +40,208 @@ function Friend() {
-                    activity you’d like to do together. It can be just a 15 min
-                    online coffee chat or any other entertainment to your taste.
-                    For inspiration, we prepared a list of cool quick games you
-                    could try. Go check it out!{" "}
+                  <div className="items-center ml-2 flex">
+                    <div className="flex-col items-center">
+                      <div className="flex text-[1rem]">
+                        <div className="text-white"> Email: </div>
+                        <div className="ml-2 text-highlighted-button-blue">
+                          {" "}
+                          {friendData.email}{" "}
+                        </div>
+                      </div>
+
+                      <div className="flex text-[1rem] text-white">
+                        <div className="text-white"> Discord: </div>
+                        <div className="ml-2 text-highlighted-button-blue">
+                          {" "}
+                          {friendData.discord}{" "}
+                        </div>
+                      </div>
+
+                      <div className="mt-[1rem] text-[1rem] text-white">
+                        Contact your friend of the week and choose together a
+                        quick activity you’d like to do together. It can be just
+                        a 15 min online coffee chat or any other entertainment
+                        to your taste. For inspiration, we prepared a list of
+                        cool quick games you could try. Go check it out!{" "}
+                      </div>
+                    </div>
                   </div>
                 </div>
               </div>
             </div>
           )}
 
           <div className="w-1/3 h-[1080px] bg-bkg-purple flex-shrink-0">
-            <div className="text-white mt-20 text-center text-[32px] font-bold">
-              Suggested Activities
-            </div>
-
-            <div className="grid gap-16 justify-center mt-12">
+            {notPaired ? (
+              <div className="text-white mt-[2.19rem] text-center text-[1.25rem] font-bold"> 
+                Suggested Activities for next week
+              </div>
+            ) : (
+              <div className="text-white mt-[2.19rem] text-center text-[1.25rem] font-bold">
+                Suggested Activities to do with {friendData.name}
+              </div>
+            )}
+              
+            <div className="grid gap-[3.5rem] justify-center mt-[2.5rem]">
               <Link to="https://skribbl.io">
                 <button
-                  className="w-72 h-20 relative flex flex-col items-center justify-center w-72 h-20 rounded-[27px] text-white text-lg font-bold bg-cover bg-center focus:border-4 focus:border-white"
+                  className="w-[15.3125rem] h-[3.5rem] w-72 h-20 rounded-[1.25rem] text-white bg-cover bg-center focus:border-4 focus:border-white"
```

Comment:
This is a prime example of something you shouldn't be doing. You don't want this random numbers everywhere. If the design uses them, the design needs to change. The difference between 15rem and 15.3rem is not noticeable and creates a codebase of chaos.

Also, you should not be using custom sizes, as I mentioned in the other PR. The default tailwind sizes should be more than enough to cover all of these scenarios. If you are not sure which default size to use, talk to your designer and figure out one, but don't add this.

PS: The CSS that figma generates is extremely bad, if you are following any of that, don't.